### PR TITLE
update polkadot-launch

### PR DESCRIPTION
**Pull request by apopiak**
Running this with a different `config.json` locally, so pls try with the one in this PR.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -13,17 +13,30 @@
...
 	},
 	"parachains": [
 		{
 			"bin": "./target/release/statemint",
 			"chain": "statemint-dev",
 			"id": "1000",
-			"wsPort": 9988,
-			"port": 31200,
 			"balance": "1000000000000000000000",
-			"flags": ["--alice", "-laura=debug", "--force-authoring", "--", "--execution=wasm"]
```
Comment (riusricardo) [inline config.json:26]: Any reason for removing the old flags?

Comment (shawntabrizi) [reply config.json:26]: I added `--alice` and `--force-authoring` to polkadot launch itself.

`-laura=debug` you should probably keep

Comment (riusricardo) [reply config.json:26]: Cool! thanks @shawntabrizi

File: config.json [hunk 2]
```diff
@@ -13,17 +13,30 @@
...
+			"nodes": [
+				{
+					"name": "alice",
+					"wsPort": 9988,
+					"port": 31200,
+					"flags": ["--", "--execution=wasm"]
```
Comment (apopiak) [inline config.json:37]:
```suggestion
					"flags": ["-laura=debug", "--", "--execution=wasm"]
```

File: config.json (hunk 1 above)
Comment (apopiak) [reply config.json:26]: jep, re-added aura debug flag

**Review by riusricardo — APPROVED** [review]
Works well for me :+1:

**Review by JesseAbram — APPROVED** [review]

**Comment by apopiak** [comment]
we'll want to investigate just pulling it in as a dependency or submodule IMO

**Pull request merged**